### ISSUE TITLE
Java array formatting

### ISSUE THREAD

**Issue by paintparty** [eerohele/pp#7]
Found an edge case hiding in a corner ...

```Clojure
(clojure.pprint/pprint (into-array '(1 2 3)))
;; Prints:
;; [1, 2, 3]


(me.flowthing.pp/pprint (into-array '(1 2 3)))
;; Prints:
;; #object["[Ljava.lang.Long;" 0x346690d6 "[Ljava.lang.Long;@346690d6"]
```

**Comment by eerohele** [comment]
Thanks for reporting. I'll fix that.

**Comment by paintparty** [comment]
:pray:

**Comment by eerohele** [comment]
Interesting that `clojure.pprint/pprint` prints Java arrays differently from `prn`:

```clojure
user=> (prn (byte-array [(int \a) (int \b) (int \c)]))
#object["[B" 0x77380939 "[B@77380939"]
nil
user=> (prn (into-array [1 2 3]))
#object["[Ljava.lang.Long;" 0x46ceabe5 "[Ljava.lang.Long;@46ceabe5"]
nil
```

**Comment by eerohele** [comment]
It is also interesting that `clojure.pprint/pprint` prints commas in arrays, but not in Clojure colls:

```clojure
user=> (clojure.pprint/pprint (into-array [1 2 3]))
[1, 2, 3]
user=> (clojure.pprint/pprint [1 2 3])
[1 2 3]
```

I don't know that I want to pp to have that inconsistency.

**Comment by eerohele** [comment]
This should be fixed in 2024-11-13.77. Can you give it a try and let me know whether it's fixed for you?

**Comment by paintparty** [comment]
Works great on my side. Thanks for the quick fix!
